### PR TITLE
fix url

### PR DESCRIPTION
**Pull request by Arhell**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -92,9 +92,9 @@ If you want, you can organize your templates in one or more folder.
 **How to write the template ?**
 -------------------------------
 
-The `appy framework <https://appyframework.org>`_ is used to interpret the template and render it using the context.
+The `appy framework <http://appyframework.org>`_ is used to interpret the template and render it using the context.
```
Comment (ale-rt) [inline README.rst:95]:
```suggestion
The `appy framework <https://appyframe.work/tool/public>`_ is used to interpret the template and render it using the context.
```

File: docs/user.rst [hunk 2]
```diff
@@ -25,9 +25,9 @@ If you want, you can organize your templates in one or more folder.
 **How to write the template ?**
 -------------------------------
 
-The `appy framework <https://appyframework.org>`_ is used to interpret the template and render it using the context.
+The `appy framework <http://appyframework.org>`_ is used to interpret the template and render it using the context.
 
-You can find a `documentation <https://appyframework.org/podWritingTemplates.html>`_ explaining the syntax that can be used.
+You can find a `documentation <http://appyframework.org/podWritingTemplates.html>`_ explaining the syntax that can be used.
```
Comment (ale-rt) [inline docs/user.rst:30]:
```suggestion
You can find a `documentation <ttps://appyframe.work/13>`_ explaining the syntax that can be used.
```

File: setup.py [hunk 3]
```diff
@@ -12,7 +12,7 @@
 setup(
     name='collective.documentgenerator',
     version='3.30.dev0',
-    description="Desktop document generation (.odt, .pdf, .doc, ...) based on appy framework (https://appyframework.org) and OpenOffice/LibreOffice",
+    description="Desktop document generation (.odt, .pdf, .doc, ...) based on appy framework (http://appyframework.org) and OpenOffice/LibreOffice",
```
Comment (ale-rt) [inline setup.py:15]:
```suggestion
    description="Desktop document generation (.odt, .pdf, .doc, ...) based on appy framework (https://appyframe.work/tool/public) and OpenOffice/LibreOffice",
```

**Review by ale-rt — CHANGES_REQUESTED** [review]

File: README.rst (hunk 1 above)
Comment (Arhell) [reply README.rst]: Done thanks

File: docs/user.rst [hunk 4]
```diff
@@ -25,9 +25,9 @@ If you want, you can organize your templates in one or more folder.
 **How to write the template ?**
 -------------------------------
 
-The `appy framework <https://appyframework.org>`_ is used to interpret the template and render it using the context.
+The `appy framework <http://appyframework.org>`_ is used to interpret the template and render it using the context.
```
[the remark this answers is not in the document]
Comment (Arhell) [reply docs/user.rst]: Done thanks

File: docs/user.rst (hunk 2 above)
Comment (Arhell) [reply docs/user.rst]: Done thanks

File: setup.py (hunk 3 above)
Comment (Arhell) [reply setup.py]: Done thanks

**Pull request merged**